stdlib/auth/src/main/ballerina/auth/jwt_supported_auth_provider_util.bal

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/config;
+import ballerina/crypto;
+import ballerina/internal;
+import ballerina/log;
+import ballerina/runtime;
+import ballerina/system;
+import ballerina/time;
+
+# Represents JWT configurations.
+#
+# + issuer - Identifier of the token issuer
+# + audience - Identifier of the token recipients
+# + clockSkew - Time in seconds to mitigate clock skew
```

Comment:
Why have we documented this field "clockSkew"? Its not defined inside the "InferredJwtAuthProviderConfig" record